pkg/project/project.go

```diff
@@ -331,32 +331,264 @@ func Write(folder, project string) error {
 
 func generateProjectName() string {
 	adjectives := []string{
-		"aged", "ancient", "autumn", "billowing", "bitter", "black", "blue", "bold",
-		"broad", "broken", "calm", "cold", "cool", "crimson", "curly", "damp",
-		"dark", "dawn", "delicate", "divine", "dry", "empty", "falling", "fancy",
-		"flat", "floral", "fragrant", "frosty", "gentle", "green", "hidden", "holy",
-		"icy", "jolly", "late", "lingering", "little", "lively", "long", "lucky",
-		"misty", "morning", "muddy", "mute", "nameless", "noisy", "odd", "old",
-		"orange", "patient", "plain", "polished", "proud", "purple", "quiet", "rapid",
-		"raspy", "red", "restless", "rough", "round", "royal", "shiny", "shrill",
-		"shy", "silent", "small", "snowy", "soft", "solitary", "sparkling", "spring",
-		"square", "steep", "still", "summer", "super", "sweet", "throbbing", "tight",
-		"tiny", "twilight", "wandering", "weathered", "white", "wild", "winter", "wispy",
-		"withered", "yellow", "young"}
+		"aged",
```

Comment:
Adding more adjectives and nouns for random project names, and formatting list to make sorting easier.